server/src/BitBakeDocScanner.ts

```diff
@@ -42,18 +42,13 @@ const variableInfosOverrides: Record<string, VariableInfosOverride> = {
   }
 }
 
-const variablesFilePath = 'doc/bitbake-user-manual/bitbake-user-manual-ref-variables.rst'
```

Comment:
Can we get rid of the path to "bitbake folder settings" now?

Reply:
no, it gets included into the python's path in order to give information about bitbake's libraries (such as bb and d). And it is about to be greatly improved

But maybe there is a way to obtain it from the environment script file? I still hope to find one.